src/cond.d

```diff
@@ -25,6 +25,20 @@ import ddmd.root.outbuffer;
 import ddmd.tokens;
 import ddmd.visitor;
 
+private __gshared Identifier idUnitTest;
+private __gshared Identifier idAssert;
+
+static this()
```

Comment:
I have same question, but currently many `__gshared` variables are intialized in `static this()`...
* `Identifier.stringtable` is initialized from `static this()` in `Token`, via `Identifier.initTable()`
* `traitsStringTable` in `traits.d` is initialized in `static this()`.
* ...